app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -31,7 +31,7 @@ class MainActivity : ComponentActivity() {
             FoodAppTheme {
                 Box (modifier = Modifier
                     .fillMaxSize()
-                    .background(Color.LightGray)) {
+                    .background(Color.Blue)) {
```

Comment:
Color selection is up to the group's UX designer.  This should only be changed in a code review if there is an accessibility issue that is being addressed, like contrast.  Is that the case here?  If so, cite the contrast differences.